test/support/builders/pages/EndDateQuestionPageBuilder.scala

```diff
@@ -28,7 +29,8 @@ object EndDateQuestionPageBuilder {
   val aEndDateQuestionPage: EndDateQuestionPage = EndDateQuestionPage(
     taxYear = taxYearEOY,
     benefitType = JobSeekersAllowance,
+    titleFirstDate = aStateBenefitsUserData.claim.get.startDate,
     sessionDataId = UUID.randomUUID(),
-    form = new FormsProvider().endDateYesNoForm(taxYearEOY)
+    form = YesNoForm.yesNoForm("", Seq.empty)
```

Comment:
It is better to put "default.error" rather than empty string. This will help later if you have problems and tests can give more meaningful error.